packages/ui/src/components/PieChart/index.tsx

```diff
@@ -81,6 +81,23 @@ export const PieChart = ({
     [emptyLegend],
   )
 
+  const localColors = Object.keys(colors.other.data.charts)
+    .filter(key => !['success', 'danger'].includes(key))
+    .sort((a, b) => {
+      if (Number(a.replace('data', '')) < Number(b.replace('data', ''))) {
+        return -1
+      }
+      if (Number(a.replace('data', '')) > Number(b.replace('data', ''))) {
```

Comment:
To avoid duplicating:
```suggestion
      const numberA = Number(a.replace('data', ''))
      const numberB = Number(b.replace('data', ''))
      if (numberA < numberB) {
        return -1
      }
      if (numberA > numberB) {
```

Reply:
@QuiiBz thanks for feedback, I simplified it a bit more so no need for variable now 😛